Src/MoneyFox.Ui/Views/Settings/SettingsViewModel.cs

```diff
@@ -1,18 +1,32 @@
 namespace MoneyFox.Ui.Views.Settings;
 
+using System.Collections.ObjectModel;
 using System.Globalization;
+using AutoMapper;
 using Core.Common.Settings;
 using Domain;
+using MediatR;
+using MoneyFox.Core.Queries;
+using MoneyFox.Ui.Views.Accounts.AccountModification;
 
 internal sealed class SettingsViewModel : BasePageViewModel
 {
     private readonly ISettingsFacade settingsFacade;
 
+    private readonly IMediator mediator;
+
+    private readonly IMapper mapper;
+
     private CurrencyViewModel selectedCurrency = null!;
 
-    public SettingsViewModel(ISettingsFacade settingsFacade)
+    private AccountViewModel? selectedAccount = null;
```

Comment:
If you use the AccountPicker control, you can use the `AccountPickerViewModel`. That loads a lot less fields and is more specific. If you can't use that, then I think it would make sense to make a dedicated one here too.

Reply:
I have used a dedicated VM as suggested.